packages/victory-brush-container/src/brush-helpers.ts

```diff
@@ -452,7 +451,11 @@ const Helpers = {
         : targetProps.defaultBrushArea;
     const defaultBrushHasArea =
       defaultBrushArea !== undefined && defaultBrushArea !== "none";
-    const mutatedProps = { isPanning: false, isSelecting: false };
+    const mutatedProps: {
```

Comment:
is it preferred to type inline like this or to create a separate type here? is there a general rule of thumb?

Reply:
Normally, inline objects like this already have an inferred type, which is fine.  In this case, though, we needed to explicitly type because there's an extra prop that we might set later.  But the concept is the same -- the type has the same scope as the object, which is fine.

Reply:
I see! got it, thanks 🙏